src/Backend/API.ts

```diff
@@ -35,47 +34,53 @@ let globalBackendMap: BackendMap = {};
 // List of Executor extensions registered
 let globalExecutorArray: Executor[] = [];
 
-function backendRegistrationApi() {
```

Comment:
I changed this name from `backendRegistraitionAPI` to `API` because it represents general API exports from ONE-vscode. Actually it contains 'executor registration API' too.